ext/strings.go

```diff
@@ -44,6 +57,46 @@ import (
 //	'hello'.charAt(5)  // return ''
 //	'hello'.charAt(-1) // error
 //
+// # Format
+//
+// Returns a new string with substitutions being performed, printf-style.
+// The valid formatting clauses are:
+//
+// `%s` - substitutes a string. This can also be used on lists, maps, bytes, Duration and Timestamp.
```

Comment:
Are enums supported? (I don't see tests either way)

Reply:
Since enums get mapped to integers, I'm not sure that we would expect anything different for enum vs. `int`. This makes rendering enums as human-readable strings difficult without introducing support for strong enums, at least at runtime.

Reply:
@DangerOnTheRanger Maybe just a test here showing that enums are not supported?

Reply:
Right, I'll add a test for that.